runtime/wasm/src/module/mod.rs

```diff
@@ -1190,6 +1218,11 @@ impl ModuleImportResolver for ModuleResolver {
             // log.log
             "log.log" => FuncInstance::alloc_host(signature, LOG_LOG),
 
+            "arweave.transactionData" => {
+                FuncInstance::alloc_host(signature, ARWEAVE_TRANSACTION_DATA)
+            }
+            "box.getProfile" => FuncInstance::alloc_host(signature, BOX_PROFILE),
```

Comment:
One small inconsistency I'm noticing here: in their docs, 3box and Arweave have endpoints like `/profile/:id` and `/tx/:id/data`. They refer to to them as "Get Profile" and "Get Transaction Data". I think we should settle on either `box.getProfile`, `arweave.getTransactionData` or `box.profile` and `arweave.transactionData`. I like the latter because it's shorter and less verb-y.

Reply:
Renamed to just `box.profile`.